workflows/theiacov/updates/wf_ncbi_scrub_pe.wdl

```diff
@@ -20,7 +20,8 @@ workflow dehost_pe {
     input:
       samplename = samplename,
       read1 = ncbi_scrub_pe.read1_dehosted,
-      read2 = ncbi_scrub_pe.read2_dehosted
+      read2 = ncbi_scrub_pe.read2_dehosted,
+      target_organism = "Severe acute respiratory syndrome coronavirus 2"
```

Comment:
For NCBI scrub workflows can we set this to be a workflow input and not hard coded on the task level?